Backoffice/AuthorizeStatusChange/AuthorizeStatusChangeInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace OpenOrchestra\Backoffice\AuthorizeStatusChange;
+
+use OpenOrchestra\ModelInterface\Event\StatusableEvent;
+
+/**
+ * Interface AuthorizeStatusChangeInterface
+ */
+interface AuthorizeStatusChangeInterface
+{
+    /**
+     * @param StatusableEvent $event
+     *
+     * @return bool
+     */
+    public function isGranted(StatusableEvent $statusableEvent);
```

Comment:
The isGranted method should not accept a statusableEvent but a StatusableInterface and a StatusInterface in the signature